src/main/io/rcdevice.c

```diff
@@ -285,9 +285,8 @@ static rcdeviceResponseParseContext_t* getWaitingResponse(timeUs_t currentTimeUs
 
 void rcdeviceReceive(timeUs_t currentTimeUs) 
 {
-    UNUSED(currentTimeUs);
     rcdeviceResponseParseContext_t *respCtx = NULL;
-    while ((respCtx = getWaitingResponse(millis())) != NULL && serialRxBytesWaiting(respCtx->device->serialPort)) {
+    while ((respCtx = getWaitingResponse(currentTimeUs / 1000)) != NULL && serialRxBytesWaiting(respCtx->device->serialPort)) {
```

Comment:
`micros() / 1000` is not equivalent to `millis()` - micros will roll over a bit after 1 hour, breaking timeout logic